### ISSUE TITLE
Massive GC Lag

### ISSUE THREAD

**Issue by NanakoAC** [discordia-space/CEV-Eris#2475]
For reasons unknown, the garbage collector is accumulating millions of things 

https://images-ext-2.discordapp.net/external/BI3KjXYHjpe_9LOarXzZOg74tZkOjfnGdTLekiZ31d4/https/i.imgur.com/Tjkw4ta.png

This is almost certainly the cause of server lag

**Comment by NanakoAC** [comment]
I did some asking around and a few people looked through our recent commits, their best suggestion was the coords objects from this PR https://github.com/discordia-space/CEV-Eris/pull/2250/files

**Comment by NanakoAC** [comment]
We have not yet found the cause of this, but debug verbs were implemented to figure out whats going on when it happens again

so far it has not happened again, as far as i'm aware.

**Comment by ACCount12** [comment]
It did happen again, resulting in a server meltdown. Here's the list of GC'd crap at the moment:
```
/obj/screen/openspace_overlay: 128314
/obj/screen/plane_master/floor: 102610
/obj/screen/plane_master/game_world: 102610
/obj/screen/plane_master/lighting: 102610
/obj/screen/plane_master/parallax_white: 102610
/obj/screen/plane_master/open_space_plane: 102610
/datum/reagents: 17916
/datum/dna: 15999
/obj/item/organ/external: 8757
/datum/gas_mixture: 6276
/icon: 3831
/obj/item/organ/external/groin: 2508
/obj/item/organ/external/head: 2508
/datum/timedevent: 2296
```
From that, it does seem like the whole thing is caused by vis_contents openspace PRs: #2607, #2737, etc. It seems like those create and destroy a lot of objects all the time, and the effects likely scales with the amount of players online. At some point, the influx of those qdels becomes enough to choke the GC, and the server suffers for it.

**Comment by ACCount12** [comment]
It was supposedly fixed in #3153. The server was stress tested by 100+ active players, and it didn't break. 

If the problem resurfaces, reopen this issue.
